### PR TITLE
Fixed search functionality, search logo appearance

### PR DESCRIPTION
**Pull request by adids1221**
## Description
Fixed search functionality, search logo appearance

## Changelog
Fixed search functionality, search logo appearance

### CODE REVIEW

File: demo/src/screens/MainScreen.js [hunk 1]
```diff
@@ -241,11 +242,9 @@ class MainScreen extends Component {
         floatingPlaceholder={false}
         text70
         leadingAccessory={
-          !filterText ? (
-            <View>
-              <Icon tintColor={Colors.$iconDefault} source={Assets.icons.demo.search}/>
-            </View>
-          ) : undefined
+          <View>
```
Comment (ethanshar) [inline demo/src/screens/MainScreen.js:245]: I think you can remove the wrapping View.
Also, let's add a small margin (s2) to the icon on the right to keep it a little spaced from the text

Comment (adids1221) [reply demo/src/screens/MainScreen.js:245]: About the margin you meant for the left button the search button right?

Comment (ethanshar) [reply demo/src/screens/MainScreen.js:245]: Yes, the left icon (the one I comment about)

Comment (adids1221) [reply demo/src/screens/MainScreen.js:245]: done

Comment (ethanshar) [reply demo/src/screens/MainScreen.js:245]: I think you missed this (: ?
> I think you can remove the wrapping View.

**Review by ethanshar — APPROVED** [review]

**Pull request merged**